compiler/mio-tflite260/CMakeLists.txt

```diff
@@ -47,3 +47,10 @@ endif(NOT TensorFlowGEMMLowpSource_FOUND)
 add_library(mio_tflite260_inc INTERFACE)
 target_include_directories(mio_tflite260_inc SYSTEM INTERFACE "${TensorFlowSource_DIR}")
 target_include_directories(mio_tflite260_inc SYSTEM INTERFACE "${TensorFlowGEMMLowpSource_DIR}")
+
+file(GLOB_RECURSE SOURCES "src/*.cpp")
+
+add_library(mio_tflite260_util ${LUCI_LIBRARY_TYPE} ${SOURCES})
```

Comment:
(1) mio_tflite260_util is not luci
(2) assume mio_tflite260_util is shared, you may need to check with debian package generation.
(3) using name util was long time ago rejected by @parjong as for broad meaning. instead helper was used or you can provide another better one.

Reply:
> (1) mio_tflite260_util is not luci

:+1:

> (2) assume mio_tflite260_util is shared, you may need to check with debian package generation.

I didn't catch this thanks :)
I changed it as static!

> (3) using name util was long time ago rejected by @parjong as for broad meaning. instead helper was used or you can provide another better one.

I will use `helper` instead :)